### PR TITLE
UA-565 and UA-564

### PR DESCRIPTION
**Pull request by jonpage**
Resolving bugs affecting series and category endpoints fixing the following endpoints which were not returning data:

```
v1/category?id=32
v1/category?id=53
v1/series/siblings?id=164724
```

### CODE REVIEW

File: data/categoryRepository.go [hunk 1]
```diff
@@ -125,7 +124,6 @@ LEFT JOIN measurement_series ON measurement_series.measurement_id = data_list_me
 LEFT JOIN series ON series.id = measurement_series.series_id
 LEFT JOIN feature_toggles ON feature_toggles.name = 'filter_by_quarantine'
```
Comment (kotodharma) [inline data/categoryRepository.go]: Would it be good to get rid of the join as well, for good luck? Doesn't look like the join is being used elsewhere in query (even if it doesn't hurt). Also... there are other queries that do this kinda join to find 'filter_by_quarantine'. Just checking that the two instances you removed are the only relevant ones, and the ones that remain are ok.

Comment (jonpage) [inline data/categoryRepository.go]: Good point. I'll remove the join. The other instances of the quarantine filter should remain. We just want the category summaries to be independent of the quarantined status of the associated series.

**Comment by kotodharma** [comment]
btw, I don't have privs to merge this, so in case you were waiting for me :=)

**Comment by kotodharma** [comment]
but of course I can do the deploy

**Pull request merged**